diff --git a/dist/module.mjs b/dist/module.mjs
index 3f96a829ffb22481209256a774daf0044e7f375e..9a0d488056dddc95aaeee818df18e933732e4e99 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -1481,7 +1481,7 @@ Missing \`${theme$1.env}\` env variable, please add it to your \`.env\`.
       return;
     }
     nuxt.hook("build:before", async () => {
-      await validateLicense({ key, theme: theme$1, dir: nuxt.options.rootDir });
+      // await validateLicense({ key, theme: theme$1, dir: nuxt.options.rootDir });
     });
   }
 });
